discovery-provider/src/tasks/index_cache.py

```diff
@@ -0,0 +1,55 @@
+import logging
+import json
+from src.tasks.celery_app import celery
+from src.tasks.generate_trending import generate_trending
+
+logger = logging.getLogger(__name__)
+
+
+######## HELPER FUNCTIONS ########
+
+
+def update_trending_cache(self, db, redis, time):
+    logger.error(f"index_cache.py | Update trending cache {time}")
+    resp = generate_trending(db, time, None, 100, 0)
+    resp_json = json.dumps(resp)
+    redis_key = f"trending-{time}"
+    logger.error(redis_key)
+    redis.set(redis_key, resp_json)
```

Comment:
@hareeshnagaraj do we want these to expire? right now this is just set indefinitely

Reply:
https://redis.io/commands/setex

Reply:
ah, i figured they would be getting overwritten all the time by the task and stay updated but maybe they should

Reply:
@hareeshnagaraj okay perfect. that works. the set command will overwrite the old value